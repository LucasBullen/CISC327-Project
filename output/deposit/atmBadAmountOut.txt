Select mode: atm or agent.
Logged into atm mode.
Enter account number to deposit to.
Enter amount to deposit.
Incorrect amount format. Deposit cancelled.
Logged out.
