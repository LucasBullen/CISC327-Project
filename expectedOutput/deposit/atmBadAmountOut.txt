Select mode: atm or agent.
Logged into atm mode.
Enter number of account to deposit into.
Enter amount (in cents) to deposit.
Amount is not a proper number. Deposit cancelled.
Logged out.
